cmd/simulator/load/loader.go

```diff
@@ -133,17 +141,37 @@ func ExecuteLoader(ctx context.Context, config config.Config) error {
 		})
 	}
 
+	go func(ctx context.Context) {
+		// Start a prometheus server to expose individual tx metrics
+		server := &http.Server{
+			Addr: MetricsPort,
+		}
+
+		go func() {
+			defer func() {
+				if err := server.Shutdown(ctx); err != nil {
+					log.Error("Metrics server error: %v", err)
+				}
+				log.Info("Received a SIGINT signal: Gracefully shutting down metrics server")
+			}()
+			sigChan := make(chan os.Signal, 1)
+			signal.Notify(sigChan, syscall.SIGINT)
+			<-sigChan
+		}()
+
+		http.Handle("/metrics", promhttp.HandlerFor(reg, promhttp.HandlerOpts{Registry: reg}))
+		log.Info("Metrics Server: localhost:8082/metrics")
```

Comment:
seems this should refer to the new constant

Reply:
thanks